### PR TITLE
Prepare the 1.0.0-beta.1 release

### LINKED ISSUE #133: Prepare initial release
A lot of work has already gone into this plugin, so it's about time we get to the most important stuff - release it! 🚢 

This issue should serve as a bit of an overview for the things we need to cover before our initial release. See the tasks below:

* [x] Define scope of the initial release, i.e. which modules should be included in which state (discussion starting in https://github.com/WordPress/performance/issues/133#issuecomment-1024056958)
* [x] Decide on the version number to use for this initial release (see https://github.com/WordPress/performance/issues/133#issuecomment-1024051757)
* [x] Add release documentation (see #131)
* [x] Finalize GitHub configuration for automatic deployment (see #42)
* [x] Prepare release based on steps in the above documentation (should be a PR for this issue)

### PR DESCRIPTION
**Pull request by felixarntz**
## Summary

Fixes #133 

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

**Review by mitogh — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -67,18 +70,42 @@ Contributions welcome! There are several ways to contribute:
 **Features**
 
 * Images: Add WebP for uploads module. ([32](https://github.com/WordPress/performance/pull/32))
+* Images: Store the backup JPEG images in `_wp_attachment_backup_sizes`. ([154](https://github.com/WordPress/performance/pull/154))
```
Comment (mitogh) [inline readme.txt]: This PR was not merged.

```suggestion
```

**Review by mitogh — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (mitogh) [reply readme.txt]: - #154

File: admin/load.php [hunk 2]
```diff
@@ -339,6 +339,84 @@ function perflab_get_module_data( $module_file ) {
...
 }
 
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @param string $hook_suffix The current admin page.
+ * @since 1.0.0
```
Comment (mitogh) [inline admin/load.php:349]: Numbers here should be `1.0.0-beta.1` ?

Comment (adamsilverstein) [reply admin/load.php:349]: same question for all version numbers below

Comment (felixarntz) [reply admin/load.php:349]: @mitogh @adamsilverstein The versions in the codebase should still always be the "main" version that it applies to. This is similar in WP core and also documented in https://github.com/WordPress/performance/blob/trunk/docs/Releasing-the-plugin.md#update-the-version-number

**Review by mitogh — APPROVED** [review]

**Review by adamsilverstein — APPROVED** [review]
🚀

**Pull request merged**